firefox-ios/nimbus-features/messaging/messaging-evergreen-messages.fml.yaml

```diff
@@ -39,23 +39,11 @@ import:
             triggers:
               ON_FOURTH_LAUNCH_THIS_YEAR: "'app_cycle.foreground'|eventSum('Years', 1, 0) > 3"
 
-        - channel: developer
+        - channel: beta
           value:
             messages:
-              # Serves as an example of how survey message might be formatted
-              survey-surface-message:
-                surface: survey
-                style: SURVEY
-                trigger-if-all:
-                  - NEVER
-                text: ResearchSurface/Body.Text.v112
-                button-label: ResearchSurface/PrimaryButton.Label.v112
-                action: OPEN_URL
-                action-params:
-                  url: https://www.macrumors.com
-
               # Serves as an example of how microsurveys may look like
-              microsurvey-message:
+              home-pagemicrosurvey-message:
```

Comment:
woild this be better as `homepage-microsurvey-message`? THis reads a bit weird atm. lol

Reply:
oh you are absolutely right, 😄 , updated!